spec/javascript/containers/LocationsContainerSpec.js

```diff
@@ -17,10 +24,8 @@ describe("LocationsContainer", () => {
     });
   });
 
-  it("Should change the LocationsContainer state to include all locations", () => {
-    expect(wrapper.state('locations')).not.toEqual({
-      locations: []
-    });
+  it("Should change the LocationsContainer state to include all available locations", () => {
+    expect(wrapper.setState({ locations: [location] }))
```

Comment:
Here, setState is just a way to manually change the state of your wrapper. It should be _before_ the expect statement, which should then test something (like, that it's rendering a `LocationTile` with the expected props)